cpp/include/cudf/detail/gather.hpp

```diff
@@ -67,5 +67,20 @@ std::unique_ptr<table> gather(
   negative_index_policy neg_indices,
   rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
+/**
+ * @copydoc cudf::detail::gather(table_view const&,column_view const&,table_view
+ * const&,cudf::out_of_bounds_policy,cudf::detail::negative_index_policy,rmm::cuda_stream_view,rmm::mr::device_memory_resource*
```

Comment:
Also need to break line.

```suggestion
 * @copydoc cudf::detail::gather(table_view const&,column_view const&,table_view
 * const&,cudf::out_of_bounds_policy,cudf::detail::negative_index_policy,rmm::cuda_stream_view,rmm::mr::device_memory_resource*)
```